mkdocs.yml

```diff
@@ -13,8 +13,8 @@ theme:
   logo: img/logos/amundsen_mark_orange.svg
   favicon: 'img/logos/amundsen_mark_orange.svg'
   palette:
-    primary: 'blue'
-    accent: 'pink'
+    primary: 'deep purple'
```

Comment:
Do they only support specific color names? Any place where we can understand the options and see if we can manually enter colors by "hex code"?

Reply:
As in, I see the colors we could choose from but looking for if this supports hex codes. Our main purple hex code is `#2B1B81` and the "orange" colors is `#FF623E`

Reply:
@ttannis it supports, update the pr to use that main purple color.